testscripts/test_fs_bind.sh

```diff
@@ -498,6 +498,7 @@ main()
 			((TST_COUNT++))
 		done
 	done
+	rm -rf "${resdir}"
 	rm -rf "${sandbox}"
 	return 0
 
```

Comment:
The testscript actually leaves the logs there in case that something went wrong, deleting the directory at the test end is wrong thing to do.

Also I don't understand why deleting ```$resdir``` that is located in ```${LTPROOT}/results/fs_bind``` can
cause problem with deleting the ```${TMPDIR}/sandbox```. Can you please elaborate on that?

Also he script should not create any directories/files in the ```${LTPROOT}``` to begin with.